docs/components/MachineLearning.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_title: "Machine Learning"
+---
+
+## Overview
+                              
+Nussknacker can evaluate ML models using the Machine Learning component. Internally, the ML Enricher uses  [JPMML-Evaluator library](https://github.com/jpmml/jpmml-evaluator) to evaluate ML models. The ML Enricher is an Enterprise component of Nussknacker and requires a separate licence, contact info@nussknacker.io for more information. 
+
+Models can be either JPMML encoded or exported with H2O Mojo/Pojo. 
+Model repository can be one of the following:
+- file system
+- MLFlow registry
+- custom model registry
+                                 
+
+## Configuration
+
+Sample configuration:
+```
+components.prinzPMML {  
```

Comment:
my biggest concern is that prinzPMML is not part of our model, I mean there's no prinzPMML.jar in components/common

Reply:
Hm, jest napisane, że _The ML Enricher is an Enterprise component of Nussknacker and requires a separate licence, contact info@nussknacker.io for more information._ , to chyba sugeruje, że nie będzie tego jara tak po prostu?

Reply:
Please write comments in English. All written comments here are public and for sake of transparency please keep them understandable for non Polish viewers. With regards to original my previous comment. Mentioned sentence can be understand in two ways:
1. Component jar is provided in distribution and you need some licence file to make it works or even no licence file is need - you can try it and if you want to use it on production you need to pay
2. Jar file will be somehow send.

Reply:
Modifited to 
> The ML Enricher is an Enterprise component of Nussknacker and requires a separate licence, contact info@nussknacker.io for license terms and instructions how to obtain jar with ML Enricher.

Reply:
I cant's see this new version in code but when you change it that way it will be fine for me. @dswiecki resolve comment if it is ok for u as well